modules/admin_manual/pages/configuration/files/mimetypes.adoc

```diff
@@ -0,0 +1,222 @@
+= Mimetypes Management
+:toc: right
+:page-aliases: configuration/mimetypes/index.adoc
+
+:description: ownCloud allows you to create aliases for mimetypes and map file extensions to a mimetype. These allow administrators the ability to change the existing icons that ownCloud uses to represent certain file types and folders, as well as to use custom icons for mimetypes and file extensions which ownCloud doesn’t natively support.
+
+== Introduction
+
+{description}
+
+This is handy in a variety of situations, such as when you might want a custom audio icon for audio mimetypes, instead of the default file icon.
```

Comment:
```suggestion
This is handy in a variety of situations, e.g. if you want a custom audio icon for audio mimetypes instead of the default file icon.
```